### PR TITLE
jupyterlab-pullrequests 3.0.0

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Notes
- mostly rebuilt from grayskull
- adds self as maintainer

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,53 +1,47 @@
...
-    - nodejs
   imports:
     - jupyterlab_pullrequests
+    - jupyterlab_pullrequests.managers
   commands:
     - pip check
-    - jupyter serverextension list 2>&1 | grep -ie "jupyterlab_pullrequests.*OK"
-    - jupyter labextension list 2>&1 | grep -ie "@jupyterlab/pullrequests.*OK"
-    - jupyter lab build
+    - jupyter server extension list 2>&1 | grep -ie "jupyterlab_pullrequests.*OK"  # [unix]
```
Comment (bollwyvl) [inline recipe/meta.yaml:32]: note: we probably do need to support the legacy `serverextension` for when launched under `jupyter notebook` for the time being (https://github.com/jupyterlab/pull-requests/pull/44)

**Comment by bollwyvl** [comment]
@conda-forge/jupyterlab-pullrequests ready for review!

File: recipe/meta.yaml (hunk 1 above)
Comment (fcollonval) [reply recipe/meta.yaml:32]: Indeed it will be nice.

**Review by fcollonval — APPROVED** [review]
Thanks a lot @bollwyvl 

Closes #7

**Pull request merged**